src/common/components/floating-faq/index.scss

```diff
@@ -0,0 +1,300 @@
+.help-btn {
+  position: fixed;
+  z-index: 900;
+  right: 2rem;
+  bottom: 2rem;
+  background: $dark-sky-blue;
+  color: $white;
+  border-radius: 20px 2px 20px 20px !important;
+  padding: 10px 20px 10px 10px !important;
+
+  .help {
+    display: inline-block;
+    margin-left: 0.4rem;
+    position: relative;
+    top: 1px;
+  }
+}
+
+.faq-container {
+  position: fixed;
+  z-index: 900;
+  right: 2.1rem;
+  bottom: 5.5rem;
+  height: 75vh;
+  width: 435px;
+  border-radius: 18px;
+  overflow: hidden;
+
+  @include themify(day) {
+    background: $white-four;
+  }
+
+  @include themify(night) {
+    background: $dark-two;
+  }
+
+}
+
+.faq-welcome {
+  display: flex;
+  height: 22%;
+  border-radius: 18px 18px 0 0;
+  z-index: 999;
+  color: $white;
+
+  @include themify(day) {
+    background-image: linear-gradient(to left, $dusk-blue, $dark-sky-blue);
+  }
+
+  @include themify(night) {
+    background-image: linear-gradient(to left, $dark-indigo, $dusk-blue);
+  }
+
+  .close-btn {
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+
+    @include themify(day) {
+      background: $dusk-blue;
+      border: none;
+    }
+
+    @include themify(night) {
+      background: $dark-indigo;
+      border: none;
+    }
+  }
+}
+
+.faq-welcome-message {
+  padding: 3rem 0 0 2.5rem;
+}
+
+.faq-content-list {
+  height: 59vh;
+  margin-top: -2rem;
+  overflow-y: auto;
+  overflow-x: hidden;
+  position: absolute;
+  z-index: 999;
+  border-radius: 14px;
+  width: 435px;
+}
+
+.faq-content-list-item {
+  z-index: 999;
+  height: auto;
+  width: 405px;
+  margin-left: 1rem;
+  border-radius: 14px;
+  overflow-y: auto;
+
+  @include themify(day) {
+    background: $white-two;
+  }
+
+  @include themify(night) {
+    background: $dark;
+  }
+
+}
+
+.contact-us {
+  margin-top: 0.5rem;
+}
+
+.section-title {
```

Comment:
@MTSaeed you need to move all classes inside component top class, so these changes won't mess up with other styles that use same name.

Reply:
@MTSaeed these classes still need to be moved under top component class so it won't mess up other component styles.